types/node-rsa/index.d.ts

```diff
@@ -0,0 +1,197 @@
+// Type definitions for node-rsa 0.4
+// Project: https://github.com/rzcoder/node-rsa
+// Definitions by: Ali Taheri <https://github.com/alitaheri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+
+declare class NodeRSA {
+    /**
+     * Generate new key with length specified.
+     */
+    constructor(key?: NodeRSA.KeyBits);
```

Comment:
So if I pass in exactly one argument, what happens? Does it `Generate new key with length specified.` or `Load key from string/buffer/components.`?
It sounds like if I pass in exactly one argument, only the former load is applicable, meaning that `format` isn't optional if you're trying to `Load key from string/buffer/components.`.